rts/lra/lra-test/lra-test-basic/pom.xml

```diff
@@ -53,6 +59,8 @@
             <lra.coordinator.port>${lra.coordinator.port}</lra.coordinator.port>
           </systemPropertyVariables>
           <redirectTestOutputToFile>false</redirectTestOutputToFile>
+          <!-- temporarily disable these tests (and track it via a JBTM-3211 -->
+          <skipTests>true</skipTests>
```

Comment:
Can you elaborate a little on why these tests need to be temporarily disabled? Please be curious to not release new Narayana version with these tests disabled as it may now already introduce regressions.

Reply:
@xstefank it is described in https://issues.jboss.org/browse/JBTM-321

@mmusgrov thanks for the link - would this be better on the [LRA](https://github.com/jbosstm/narayana/tree/LRA) feature branch unless we could release without it?

Reply:
@tomjenkinson JBTM-3211 doesn't go into details neither. I am just curious about what isn't working.

Reply:
Okay I have marked it as a blocker

Reply:
It should be easy to fix (I was mainly running against the TCK and I discovered the failure when I did a full build yesterday morning) but I wanted to get some eyes on the PR. But I don't see why it needs to block this PR (after all NarayanaLRAClient effectively became internal after we pulled the java client API from the spec). I will fix it now if it is a problem.

Reply:
I agree, if NarayanaLRAClient is internal it seems fine to me to leave it on this branch. (I assume we don't have quickstarts for NarayanaLRAClient still?)

Reply:
Agreed, no need to block this PR for it. I just wanted to bring it up as I am not sure whether for instance, Helidon exposes this functionality. And there are also tests for non-root JAX-RS participants IIRC.

Reply:
I took a look and found the issue. The new RequestBuilder class I added was buggy (calling builder.path(p1).path(p2) didn't append p2 correctly). So thanks to both of you for picking up on this issue.

Also Matej Kralik is using our client so it would be a bit rude if we didn't fix asap